### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by rbeyer**
This is to add a link for our shiny new Google Calendar.  I still need individual addresses to give permissions for others to add events to the calendar.

### CODE REVIEW

**Review by jlaura — APPROVED** [review]

**Review by jessemapel — COMMENTED** [review]
We should try and keep the language as affirmative as possible. I'd change

> should be entered

to

> will be entered

File: meetings/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+As meetings are planned above, they should be entered into our
+[Google Calendar](https://calendar.google.com/calendar?cid=aXFtYmE4MmQ2MGUwdDRycGIzZ2U0MjhwbTBAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ)
+that you can subscribe to.  If you are a TSC member and need the ability to post events, send Ross your address to give permissions to.
```
Comment (jessemapel) [inline meetings/README.md]: Giving people permissions to the calendar should be addressed under the [onboarding document](https://github.com/planetarysoftware/TSC/blob/master/ONBOARDING.md)

Comment (rbeyer) [reply meetings/README.md]: There only, or both places?

Comment (jessemapel) [reply meetings/README.md]: I would say there only, the readme should be for the general viewing public.

**Review by jessemapel — APPROVED** [review]

**Pull request merged**